content/en/installation/multioverview.md

```diff
@@ -0,0 +1,40 @@
+
+## Concept
+
+A multi-node cluster is composed of at least one master node and one worker node. You can add additional nodes based on your needs (e.g. for high availability) both before and after the installation.
+
+- **Master**. A master node generally hosts the control plane that controls and manages the whole system.
+- **Worker**. Worker nodes run the actual applications deployed on them.
+
+## Scenarios
+
+A multi-node architecture is extremely useful in a variety of scenarios. A couple of examples are given as follows for your reference.
+
+### High Availability (HA)
+
+For single-node clusters, users will not be able to access data stored in the cluster once the node goes down. For highly available clusters with multiple nodes, the architecture features a failover system that will allow other nodes to take over in case of a node failure. This is extremely important to replicated apps as the scheduler can assign each replica to a different node if you have enough available ones. If a node fails, your app still functions. In other words, your services can run consistently.
+
+For more information, see HA Cluster Configuration.
+
+### Rolling Update
+
+Enterprises strive to deploy new versions of their apps so that they can consistently deliver fresh experiences to their users. To this end, developers may adopt rolling updates which allow them to update instances incrementally with zero downtime. In this process, new Pods will be scheduled on nodes with available resources. With a multi-node architecture, new releases can be distributed while not affecting any services for users.
+
+As KubeSphere features great scalability, you can customize the architecture based on your business both before and after the installation.
+
+## Why Multi-node Installation of KubeSphere
+
+If you are not familiar with Linux and network management, you may find it difficult to set up a highly-functional multi-node Kubernetes cluster. Starting from the version 3.0.0, KubeSphere uses a brand-new installer called KubeKey to replace the old ansible-based installer. Developed in Go language, KubeKey allows users to quickly deploy a multi-node architecture.
+
+For users who do not have an existing Kubernetes cluster, they only need to create a configuration file with few commands and add node information (e.g. IP address and node roles) in it after KubeKey is downloaded. With one command, the installation will start and no additional operation is needed. If users have an existing multi-node Kubernetes cluster, they only need to execute a couple of commands once the environment requirement is met.
+
+For more information about the specific step of multi-node installation, see Installing on Public Cloud and Installing on On-premises Environment.
```

Comment:
Actually, you need to elaborate on [this part](https://github.com/kubesphere/kubekey/blob/master/docs/config-example.md) of Kubekey. multi-node installation is only used for Kubekey advanced mode. Give some explanation for each parameter and field:

![image](https://user-images.githubusercontent.com/40452856/89728817-ac43dc00-da62-11ea-85e6-1dfe9f9f1f97.png)

Reply:
I thought the explanation for this configuration file will be covered in the detailed installation step so I did not put it here, but anyway I will add it.

Reply:
No, it should include the explanation, since the installation step in each individual installation guide that will reuse and refer to the detailed explanation here.